src/server/rest/v1/service/StatisticService.ts

```diff
@@ -473,36 +474,38 @@ export default class StatisticService {
   static convertToCSV(loggedUser: UserToken, transactionStatsMDB: any[], dataCategory: string, dataType: string, year: number | string, dataScope?: string): string {
     let user: User;
     let unknownUser = Utils.buildUserFullName(user, false, false);
+    const i18nManager = new I18nManager(loggedUser.locale);
```

Comment:
> I would even suggest to have a request.getI18nManager() to avoid creating a new instance each time we need some translation! What do you think?

i18nManager class shall use the singleton design pattern, in the whole e-mobility codebase. I do know if that can be part of that PR for ev-server. 
It's a pretty easy change to do that will likely have a measurable impact on the memory consumption, especially on the dashboard.

Reply:
> > I would even suggest to have a request.getI18nManager() to avoid creating a new instance each time we need some translation! What do you think?
> 
> i18nManager class shall use the singleton design pattern, in the whole e-mobility codebase. I do know if that can be part of that PR for ev-server.
> It's a pretty easy change to do that will likely have a measurable impact on the memory consumption, especially on the dashboard.

This was what Claude proposed :-)
Ok with that.